serveradmin/serverdb/models.py

```diff
@@ -38,10 +38,10 @@
 }
 
 IP_ADDR_TYPES = [
-    ('null', 'null (intern_ip must be empty and inet attributes not be used)'),
-    ('host', 'host (intern_ip must be a host /32,/128 and unique)'),
-    ('loadbalancer', 'loadbalancer (intern_ip must be a host /32,/128)'),
-    ('network', 'network (intern_ip must be a network and not overlap)'),
+    ('null', 'null: intern_ip must be empty, no inet attributes'),
+    ('host', 'host: intern_ip & inet must be an ip address and unique across all objects'),
+    ('loadbalancer', 'loadbalancer: intern_ip & inet must be an ip address'),
+    ('network', 'network: intern_ip & inet must be an ip network, not overlapping with same servertype'),
```

Comment:
If this is written English, then you might rather want to use `and` instead of `&`.

Reply:
I don't see any good reason why `and` is in preference of `&` but I also don't see any reason to discuss about this as they are effectively the same so I just adjusted this to `and`.

Reply:
In "proper" written English, the `&` is very rarely used, and just used to combine nouns for a strong relationship. Got this page with a few examples: https://english.stackexchange.com/questions/3127/when-to-use-instead-of-and Therefore, I guess `and` is a better use than `&` here. But, sure, nothing to worry about for the PR.